Back/Reciplastk.Gateway/DataAccess/ReciplastkContext.cs

```diff
@@ -27,6 +27,8 @@ public ReciplastkContext(DbContextOptions<ReciplastkContext> options)
 
     public virtual DbSet<Test> Tests { get; set; }
 
+    public virtual DbSet<Weightcontrol> Weightcontrols { get; set; }
+
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 #warning To protect potentially sensitive information in your connection string, you should move it out of source code. You can avoid scaffolding the connection string by using the Name= syntax to read it from configuration - see https://go.microsoft.com/fwlink/?linkid=2131148. For more guidance on storing connection strings, see https://go.microsoft.com/fwlink/?LinkId=723263.
         => optionsBuilder.UseNpgsql("Host=reciplastk.cty0a2ewmeb5.us-east-1.rds.amazonaws.com;Database=Reciplastk;Username=postgres;Password=Admin123");
```

Comment:
For now you should change the database conection, this will be fixed soon for a permanent solution using configuration connection strings.
```suggestion
        => optionsBuilder.UseNpgsql("Host=Localhost;Database=Reciplastk;Username=postgres;Password=Admin123");
```